ocaml/tests/test_daemon_manager.ml

```diff
@@ -138,19 +136,19 @@ let test_timeout_fail () =
       error = Failure "stop failed";
       time_until_stopped = 5.0;
     };
-  assert_raises (Failure "stop failed")
+  Alcotest.check_raises "does not stop within timeout"
+    (Failure "stop failed")
     (fun () -> Mock_manager.with_daemon_stopped ~timeout:2.0 (fun () -> ()));
-  assert_equal !Mock_daemon.times_called_start 0;
-  assert_equal !Mock_daemon.times_called_stop 1
+  Alcotest.(check int) "times_called_start" !Mock_daemon.times_called_start 0;
+  Alcotest.(check int) "time_until_stopped" !Mock_daemon.times_called_stop 1
 
```

Comment:
I notice that lines 142-143 are repeated 7 times with different numbers, is it worth making an alias function?

Reply:
Sounds good, done